Parse/src/main/java/com/parse/ParsePush.java

```diff
@@ -96,6 +98,18 @@ public Builder expirationTimeInterval(Long expirationTimeInterval) {
         return this;
       }
 
+      public Builder pushTime(Long pushTime) {
+        if (pushTime != null) {
+          long now = System.currentTimeMillis() / 1000;
+          long twoWeeks = 60*60*24*7*2;
+          checkArgument(pushTime > now, "Scheduled push time can not be in the past");
+          checkArgument(pushTime < now + twoWeeks, "Scheduled push time can not be more than " +
```

Comment:
Yes see [here](http://docs.parseplatform.org/rest/guide/#scheduling-pushes). Same code is in iOS version of this feature so it should be ok